setup.cfg

```diff
@@ -34,3 +34,4 @@ console_scripts =
     fix-html = mbtools.fix_html:main
     inject-quiz-uuids= mbtools.inject_quiz_uuids:main
     generate-quiz-csv = mbtools.generate_quiz_csv:main
+    inject-ib-ids = mbtools.inject_ib_ids:main
```

Comment:
Should we call this `inject-ib-uuids` to be consistent with `inject-quiz-uuids`? If so, let's rename the python file too.

Reply:
Agreed, I think the current tongue-tying alliteration is also too much 😂